### PR TITLE
Clarified third party reporting

### PR DESCRIPTION
**Pull request by dhairyya**
Updated third party reporting document as per few support tickets I saw

### CODE REVIEW

**Comment by dhairyya** [comment]
@chrisda could you please approve this

**Comment by dhairyya** [comment]
@MSFTTracyP could you please approve this for me?

File: microsoft-365/security/office-365-security/user-submission.md [hunk 1]
```diff
@@ -174,22 +174,24 @@ When you're finished on the **User submissions** page, click **Save**. To restor
...
 
-The only requirement is that the original messages are included as uncompressed .EML or .MSG attachments in messages that are sent to user submissions mailbox. In other words, don't just forward the original messages to the user submissions mailbox.
+When **Microsoft Outlook Report Message button** is **Off** ![Toggle off.](../../media/scc-toggle-off.png) the following settings are available on the **User reported settings** page:
+
+- **Use Microsoft's integrated Outlook reporting experience** section:
+
+  **Use this custom mailbox to receive user reported messages**: Select this option and enter the email address of an existing Exchange Online mailbox to read end user reports made using the third-party message reporting tools. _This reports won't be submitted to Microsoft but will be shown on user reported page under Actions & Submissions with result - Not Submitted to Microsoft. The result means that the end user reports made from third party reporting tools were not made to Microsoft and thus there is no result available._
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/user-submission.md]:
```suggestion
  **Use this custom mailbox to receive user reported messages**: Select this option and enter the email address of an existing Exchange Online mailbox to read end-user reports made using the third-party message reporting tools. _These reports won't be submitted to Microsoft but will be shown on the user reported page under Actions & Submissions with the result - Not Submitted to Microsoft. The result means that the end-user reports made from third-party reporting tools were not made to Microsoft and thus there is no result available._
```

File: microsoft-365/security/office-365-security/user-submission.md [hunk 2]
```diff
@@ -174,22 +174,24 @@ When you're finished on the **User submissions** page, click **Save**. To restor
...
+  
+The only requirement while Microsoft is reading from that mailbox is that the original messages are included as uncompressed .EML or .MSG attachments in messages that are sent to user submissions mailbox. In other words, don't just forward the original messages to the user submissions mailbox.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/user-submission.md]:
```suggestion
The only requirement while Microsoft is reading from that mailbox is that the original messages are included as uncompressed .EML or .MSG attachments in messages that are sent to the user submissions mailbox, so you should not just forward the original messages to the user submissions mailbox.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dhairyya, kindly commit the suggested changes. Many thanks.

**Pull request merged**